Framework/include/QualityControl/MonitorObject.h

```diff
@@ -141,6 +143,9 @@ class MonitorObject : public TObject
   // tells Merger to create an object with data from the last cycle only on the side of the complete object
   bool mCreateMovingWindow = false;
 
+  void releaseObject();
+  void cloneAndSetObject(const MonitorObject&);
+
   ClassDefOverride(MonitorObject, 12);
```

Comment:
you must increase the class version otherwise ROOT can't do the schema evolution.
not super important now that we don't store MO in the QCDB but still...